sensor_config.tf

```diff
@@ -17,6 +17,9 @@ module "sensor_config" {
   sensor_management_interface_name             = "eth0"
   sensor_monitoring_interface_name             = "eth1"
   sensor_health_check_probe_source_ranges_cidr = [local.azure_lb_health_check_probe_ip]
+  sensor_health_check_http_port                = "41080"
```

Comment:
I see the literal value `41080` across the module. Did we mean to remove these now that a variable is available?